### PR TITLE
Erratic dialogs(#191), "align" in right menu(#180)

### LINKED ISSUE #180: "align" in right menu
Common use of "align" is to align several selected items in some way. I think this button shoud be renamed to "layout".

### LINKED ISSUE #191: erratic dialogs
image
image
image
image
image
image

### PR DESCRIPTION
**Pull request by mateuszkp96**
*This resolves #191 
*This resolves #180

### CODE REVIEW

**Comment by PPiedel** [comment]
Please put some photos of how it looks after the changes

**Comment by lciolecki** [comment]
Plase use 2 spaces instead tab / 4 spaces.

File: ui/client/components/RouteLeavingGuard.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        const {modalVisible, lastLocation} = this.state
+        return (
+            <>
+                <Prompt
+                    when={when}
+                    message={(location, action) => this.handleBlockedNavigation(location, action)}/>
+                <PreventExitDialog
+                    visible={modalVisible}
+                    onCancel={this.closeModal}
+                    onConfirm={this.handleConfirmNavigationClick}/>
```
Comment (lciolecki) [inline ui/client/components/RouteLeavingGuard.js]: Maybe better formatting, I mean that you can put /> in new line ;)

**Comment by mateuszkp96** [comment]
image
compare versions

**Comment by mateuszkp96** [comment]
image
image

**Comment by mateuszkp96** [comment]
image
Disabled archive button when process is running

**Pull request merged**